great_expectations_cloud/agent/event_handler.py

```diff
@@ -111,6 +111,17 @@ def parse_event_from(cls, msg_body: bytes) -> Event:
 
         return event
 
+    @classmethod
+    def parse_event_from_dict(cls, msg_body: dict[str, Any]) -> Event:
```

Comment:
Can you modify the existing tests for `parse_event_from` in `test_event_handler.py` to use this method instead (in addition to removing `parse_event_from`)?